src/ui/MessageItemMenu/index.tsx

```diff
@@ -38,7 +38,7 @@ interface Props {
   onMoveToParentMessage?: () => void;
 }
 
-export default function MessageItemMenu({
+export function MessageMenu({
```

Comment:
Is there a specific reason for changing the component name differently from the file name?

Reply:
I tried to match the prop names and the component names
* renderMessageMenu - MessageMenu
* renderEmojiMenu - MessageEmojiMenu

And we were using `export default` so I thought it wouldn't make the backword issue.
But we can make it back if it's not good practice

Reply:
I see~ generally, it's always good to ensure that the file name and the component name match, especially for file navigation or stack trace tracking
If you feel that changes are needed, it's a good idea to leave a TODO marker and consider addressing it in v4